traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -86,7 +86,7 @@ func WriteRespRaw(w http.ResponseWriter, r *http.Request, v interface{}) {
 		return
 	}
 	w.Header().Set("Content-Type", "application/json")
-	w.Write(bts)
+	w.Write(append(bts, '\n'))
```

Comment:
I believe this is causing a unit test to fail in this package ->

```
--- FAIL: TestReadHandler (0.00s)
    /Users/mhoppa509/go/src/github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api/shared_handlers_test.go:187: Expected body {"response":[{"ID":1}]} got {"response":[{"ID":1}]}
```

Reply:
whoops :P